### ISSUE TITLE
Add pagination instead of limiting records 

### ISSUE THREAD

**Issue by ahmadabdelhalim** [emad-elsaid/rubyfunctions#48]
- **Problem** 
The records are limited to a fixed number. 

- **Solution**
Add pagination using the Pagy gem. (I chose it because it's faster than other gems available)

**Comment by emad-elsaid** [comment]
There is pagination implemented in user page to show more of his functions
and saved functions, that could be also used in the timelines controller

**Comment by ahmadabdelhalim** [comment]
Yeah, didn't notice that. I can add it to the timelines controller as well.

**Comment by emad-elsaid** [comment]
@ahmadabdelhalim nice, I'll wait for a PR then :smiley:
